doc/essentials/EnergyPlusEssentials.tex

```diff
@@ -0,0 +1,2260 @@
+It was developed as a simulation engine, and many graphical user interfaces
+utilize it.
+
+\subsection{Documentation and Other Resources}
+
+The EnergyPlus documentation is currently included in the installation
+in the ``Documentation'' folder as PDFs. It is also available as
+\href{https://energyplus.net/documentation}{PDFs online} and as \href{https://bigladdersoftware.com/epx/docs/}{browsable HTML}.
+Using search in the documentation is critical to finding the information
+needed. Some of the documentation PDFs are very large, so searching
+is a good way to find specific information about a topic or an input
+object. The documentation includes:
+\begin{itemize}
+\item QuickStart Guide: Contains a brief high-level overview of EnergyPlus
+and get you up and running quickly with the program
+\item Input Output Reference: Contains a thorough description of the various
+input and output files related to EnergyPlus, the format of these
+files, and how the files interact and interrelate.
+\item Output Details and Examples: Contains details on output from EnergyPlus.
+It also addresses the reference data sets that are included.
+\item Auxiliary Programs:~Contains information for the auxiliary programs
+that are part of the EnergyPlus package. For example, this document
+contains the user manual for the Weather Converter program, descriptions
+on using Ground Heat Transfer auxiliary programs with EnergyPlus,
+HVAC Template descriptions, and other assorted topics.
+\item Engineering Reference: Provides more in-depth knowledge into the theoretical
+basis behind the various calculations contained in the program including
+algorithm descriptions.
+\item Application Guide for EMS: Contains information useful to use the
+advanced feature of EnergyPlus: Energy Management System tweaks. The
+Erl language is described, and examples for use are given.
+\item External Interface(s) Application Guide: Contains information specific
+to using the external interface feature of EnergyPlus to connect other
+simulation systems.
+\item Plant Application Guide: Details the methods for simulating real chilled
+and hot water plant systems within EnergyPlus.
+\item Using EnergyPlus for Compliance Guide: Contains information specific
+to using EnergyPlus in compliance and standard rating systems.
+\item Tips \& Tricks for Using EnergyPlus: Contains short tips and tricks
+for using various parts of EnergyPlus. 
+\end{itemize}
+The example files are a great resource to learn more about how to
+use EnergyPlus The example files are in the directory ExampleFiles.
+
+\section{The EnergyPlus Ecosystem }
+
+\subsection{Current Interfaces }
+
+EnergyPlus is often used directly using the text file input (IDF or
+epJSON) and various output file formats along with the utilities that
+come with the installation package. More information on that can be
+found in the \secref{Using-EnergyPlus} section. In addition, EnergyPlus
```

Comment:
This secref is rendering as "in the section §3 section".  Might want to change this to just `found in \secref{Using-EnergyPlus}`.